roles/ceilometer-common/tasks/main.yml

```diff
@@ -14,7 +14,7 @@
         owner=ceilometer group=ceilometer
 
 - name: ceilometer config
-  template: src=etc/ceilometer/{{ item }} dest=/etc/ceilometer/{{ item }} mode=0644
+  template: src=etc/ceilometer/{{ item }} dest=/etc/ceilometer/{{ item }} mode={{ 0644 if 'policy.json' in item else 0640 }} owner=ceilometer group=ceilometer
```

Comment:
They do, it's sad, and we could break them all out into a mostly duplicate task for legibility sake.